tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/NLogTests.cs

```diff
@@ -41,6 +41,7 @@ public NLogTests(ITestOutputHelper output)
             : base(output, "LogsInjection.NLog")
         {
             SetServiceVersion("1.0.0");
+            // SetEnvironmentVariable("DD_INTERNAL_SKIP_METHOD_BODY_ENABLED", "0");
```

Comment:
```suggestion
```

Reply:
I am going through these tests though so I can clean this up in it as well if you don't want to re-run CI

Reply:
removed.